### PR TITLE
pythonPackages.bleak: 0.10.0 -> 0.11.0

### PR DESCRIPTION
**Pull request by oxzi**
###### Motivation for this change
Version bump.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116731` at 9c766f6c run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build (new failures):</summary>
  <ul>
    <li><a href="https://gist.github.com/fbb69463174de7d8d10669563a508cbf">python38Packages.pyzerproc</a></li></li>
    <li><a href="https://gist.github.com/8f6739e419bf3fb6e7b2e11e0a9dbe27">python39Packages.pyzerproc</a></li></li>
  </ul>
</details>
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/57b8a3744b214cffb85e13f41f3c9db4">python38Packages.bleak</a></li></li>
    <li><a href="https://gist.github.com/fd9d72aa924342cd481e11508827d437">python39Packages.bleak</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/bleak/default.nix [hunk 1]
```diff
@@ -19,7 +21,7 @@ buildPythonPackage rec {
       --replace \"bluetoothctl\" \"${bluez}/bin/bluetoothctl\"
   '';
 
-  propagatedBuildInputs = [ txdbus ];
+  propagatedBuildInputs = [ dbus-next ];
   checkInputs = [ pytest pytestcov ];
 
   checkPhase = "AGENT_OS=linux py.test";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/bleak/default.nix:27]: This should be converted to pytestCheckHook.

Comment (oxzi) [reply pkgs/development/python-modules/bleak/default.nix]: I have updated the check inputs. Thanks.

File: pkgs/development/python-modules/bleak/default.nix [hunk 2]
```diff
@@ -19,10 +21,9 @@ buildPythonPackage rec {
       --replace \"bluetoothctl\" \"${bluez}/bin/bluetoothctl\"
   '';
 
-  propagatedBuildInputs = [ txdbus ];
-  checkInputs = [ pytest pytestcov ];
+  propagatedBuildInputs = [ dbus-next ];
 
-  checkPhase = "AGENT_OS=linux py.test";
+  checkInputs = [ pytestCheckHook pytestcov ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/bleak/default.nix:26]:
```suggestion
  checkInputs = [ pytestCheckHook pytest-cov ];
  
  pythonImportsCheck = [ "bleak" ];
```

File: pkgs/development/python-modules/bleak/default.nix [hunk 3]
```diff
@@ -1,14 +1,16 @@
-{ lib, buildPythonPackage, isPy3k, fetchPypi, bluez, txdbus, pytest, pytestcov }:
+{ lib, buildPythonPackage, isPy3k, fetchPypi
+, bluez, dbus-next, pytestCheckHook, pytestcov
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/bleak/default.nix:2]:
```suggestion
, bluez, dbus-next, pytestCheckHook, pytest-cov
```

Comment (oxzi) [reply pkgs/development/python-modules/bleak/default.nix]: Changed.

**Pull request merged**